README.md

```diff
@@ -3,9 +3,74 @@
 <!-- Nice backronym... -->
 
 Contains
-- **Fairy** a library for reading N64 ELF files (big-endian)
+- **Fairy** a library for reading relocatable MIPS ELF object files (big-endian, suitable for Nintendo 64 games)
 - **Fado** a program for generating the `.ovl`/relocation section for Zelda64 overlay files
+- **Mido** an automatic dependency file generator
 
-Compatible with both IDO and GCC.
+Compatible with both IDO and GCC (although [see below](N_B)).
 
-Format is the standard .ovl section, with the relocs divided by section. It will also print the name the associated variable if it can find it in the ELF (for IDO, this is only if it is not static, whereas GCC sometimes seems to retain all of them).
+Format is the standard "Zelda64" .ovl section, with the relocs divided by section, as used by
+- *The Legend of Zelda: Ocarina of Time* (all Nintendo 64/Gamecube/iQue releases)
+- *The Legend of Zelda: Majora's Mask* (all Nintendo 64/Gamecube releases)
+
+In theory it will also work for other Nintendo 64 games that use this system, such as *Yoshi's Story*, but has yet to be tested with these.
+
+
+## Explanation
+
+The overlay relocation sections used by Zelda64 is described [here](z64_relocation_section_format.md). Fado will produce a `.ovl` section compatible with this format, although as noted there, some compilers need persuasion to produce compatible objects.
+
+
+## How to use
+
+Compile by running `make`.
+
+A standalone invocation of Fado would look something like
+
+```sh
+./fado.elf z_en_hs2.o -n ovl_En_Hs2 -o ovl_En_Hs2_reloc.s
+```
+This takes as input the compiled object file from the C file (currently located [here](https://github.com/zeldaret/oot/blob/master/src/overlays/actors/ovl_En_Hs2/z_en_hs2.c)), the name of the overlay (`ovl_En_Hs2`) and will output an assembly file `ovl_En_Hs2_reloc.s` containing the relocation section. An example output is included in the repo [here](ovl_En_Hs_reloc.s). Fado will print information from the object file to assist with debugging, by splitting relocs by section, and for each, printing the type, offset, and associated symbol (or section if static):
```

Comment:
You could use a permalink for this actor, so this link would lead to the same state as when you make that reloc